config.lua

```diff
@@ -7,16 +7,49 @@ Config.Warnings = false -- if true it will show warnings in the console when cer
 
 Config.ChangeColor = false -- Change the color of the box of the popup text based on if it is locked or not
 Config.DefaultColor = 'rgb(19, 28, 74)' -- The default color of the box of the popup text if Config.ChangeColor is false
 Config.LockedColor = 'rgb(219 58 58)' -- The color of the box of the popup text if Config.ChangeColor is true and the door is locked
 Config.UnlockedColor = 'rgb(27 195 63)' -- The color of the box of the popup text if Config.ChangeColor is true and the door is unlocked
+Config.UseDoorLabelText = false -- Will use the LABEL field as the nui text instead of locked/unlocked
+Config.DoorDebug = false -- Enable DRAWTEXT in the world at the coords where the door 'center' is
+Config.RemoteTriggerDistance = 15.0 -- This is how far from your camera the raycast will go to try to hit something solid
+Config.RemoteTriggerMinDistance = 5.0 -- This is the minimum distance required for the raycast hit to count near a door. It will take the larger two between this and 'distance' option
 
 Config.Consumables = { ['ticket'] = 1, ['paperclip'] = 1 } -- The items will get removed once used on a door that has the item on it
 
+--[[ -- Configuration Options
+Config.DoorList['configname-identifier'] = {
+	fixText = false, -- Adjusts guess of center of door
+	textCoords = vector3(x, y, z) -- Set the text coordinates to a specific location
+	setText = true -- Use with above setting
+    	distance = 2.0, -- Max interact distance
+    	lockpick = true, -- Alows for lockpicking
+	allAuthorized = true, -- Anyone can use door
+	authorizedJobs = { ['police']=0, ['bcso']=0, ['sasp]=0 } -- Authorize door access based on job grade
+	authorizedGangs = { ['vagos']=0, ['ballas']=0 } -- Authorize door access based on gang grade
+	authorizedCitizenIDs = { 'ABC123', 'DEF456' } -- Authorize door access based on citizenid
+	items = {'item_1','item_2'} -- Authorize based on items. Must have all items in this list.
+    	doorLabel = 'Cloakroom', -- Label of room that shows up when nearby
+    	locked = true, -- Default lock state
+    	audioRemote = true, -- Play sound from the player instead of the door
+    	audioLock = {['file'] = 'metal-locker.ogg', ['volume'] = 0.6}, -- Play sound on door lock
+    	audioUnlock = {['file'] = 'metallic-creak.ogg', ['volume'] = 0.7}, -- Play sound on door unlock
+    	autoLock = 1000, -- Auto lock after this many miliseconds
+    	doorRate = 1.0,  -- Time till auto closes? Needs more testing
+	cantUnlock = true -- Set to true to not allow the player to unlock the door, only lock it. This means a script will have to trigger the unlock.
+	pickable = true, -- Can use a lockpick to unlock, only need if true
+	hideLabel = true, -- Set to true to hide the popup label, for hiding doors ;)
+    	remoteTrigger = true, -- If you want to be able to remote trigger a door with H, put this here 
+} 
+]]
+
 Config.DoorList = {
-	--- door1 for pacific opened with security card B
+Config.DoorStates = {}
```

Comment:
The Config.DoorStates are within the Config.DoorList which breaks the entire Config for everyone.